libp2p/crypto/p2pSigner.go

```diff
@@ -1,68 +1,76 @@
 package crypto
 
 import (
-	"fmt"
+	"crypto/sha256"
 
 	"github.com/ElrondNetwork/elrond-go-core/core"
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
 	crypto "github.com/ElrondNetwork/elrond-go-crypto"
-	libp2pCrypto "github.com/libp2p/go-libp2p-core/crypto"
-	"github.com/libp2p/go-libp2p-core/peer"
 )
 
 // TODO: adapt all package to use common crypto interfaces and remove this component
```

Comment:
we will not remove this file, apparently so we should remove the TODO

Reply:
removed the todo